components/Quiz.tsx

```diff
@@ -1,59 +1,56 @@
 "use client"
 
+import { FormEventHandler } from "react"
 import { Button } from "@shadcn/ui/button"
-import { Separator } from "@shadcn/ui/separator"
-import { FormEventHandler, useState } from "react"
+
 import { Typography } from "./Typography"
-import { Textarea } from "@shadcn/ui/textarea"
 
-/**
- * Don't see a need for the { isCorrect } prop now as we don't have the answer as an array
- */
-interface Props {
-  question: string
-  answer: string
+type Answer = {
+  text: string
+  isCorrect: boolean
 }
 
-function validateAnswer(answer: string) {
-  return answer.replaceAll("", "").toLowerCase()
+interface Props {
+  question: string
+  answers: Answer[]
 }
 
-export function Quiz({ question, answer }: Props) {
-  const [isCorrect, setIsCorrect] = useState(false)
-
+export function Quiz({ question, answers }: Props) {
   const handleSubmit: FormEventHandler<HTMLFormElement> = (event) => {
     event.preventDefault()
+  }
 
-    const userResponse = validateAnswer(event.currentTarget.quiz.value)
-    const correctAnswer = validateAnswer("Coursition")
+  const isMultipleCorrect = answers.filter((answer) => answer.isCorrect).length > 1
 
-    if (userResponse !== correctAnswer) return
-    setIsCorrect(true)
-  }
+  const SelectAlert = () => (
+    <>
+      {isMultipleCorrect ? (
+        <div className='absolute top-16'>
+          <Typography variant='text'>Select all that apply.</Typography>
+        </div>
+      ) : null}
+    </>
+  )
 
   return (
-    <div className="flex justify-between gap-4">
-      <div className="flex flex-col gap-3 w-1/2">
-        <Typography variant="h3">{question}</Typography>
-        <form className="contents" onSubmit={handleSubmit}>
-          <Textarea
-            name="quiz"
-            required
-            placeholder="Enter your answer here"
-            className="h-28 w-full rounded-md border-2 border-gray-300 p-2"
-            defaultValue={answer ?? ""}
-          />
+    <div className='relative justify-between gap-4 selection:flex '>
+      <div className='flex w-1/2 flex-col gap-3'>
+        <Typography variant='h3'>{question}</Typography>
+        <SelectAlert />
```

Comment:
**suggestion:** It is not necessary to move this into separate "component", it makes the code confusing without any clear value.

Reply:
Okay :)